plugin.xml

```diff
@@ -47,7 +47,7 @@
         </array>
       </config-file>
 
-      <framework src="Intercom" type="podspec" spec="~> 3.2.1" />
+      <framework src="Intercom" type="podspec" spec="~> 3.2.2" />
```

Comment:
Yes? The same change was made in https://github.com/intercom/intercom-cordova/commit/41791ea413fa3ae27400715a2feb656d074b4f25 so I copied it.